plugins/woocommerce/tests/e2e/README.md

```diff
@@ -159,23 +159,23 @@ Note that running `pnpm docker:down --filter=woocommerce` and then `pnpm docker:
 To run e2e tests in headless mode use the following command:
 
 ```bash
-pnpm -- turbo run e2e --filter=woocommerce
+pnpm run e2e --filter=woocommerce
 ```
 
 ### How to run tests in non-headless mode
 
 Tests run in headless mode by default. However, sometimes it's useful to observe the browser while running or developing tests. To do so, you can run tests in a non-headless (dev) mode:
 
 ```bash
-pnpm -- turbo run e2e:dev --filter=woocommerce
+pnpm run e2e:dev --filter=woocommerce
 ```
 
 The dev mode also enables SlowMo mode. SlowMo slows down Puppeteer’s operations. This makes it easier to see what is happening in the browser.
 
 By default, SlowMo mode adds a 50 millisecond delay between test steps. If you'd like to override the length of the delay and have the tests run faster or slower in the `-dev` mode, pass `PUPPETEER_SLOWMO` variable when running tests as shown below:
 
 ```
-PUPPETEER_SLOWMO=10 pnpm -- turbo run e2e:dev --filter=woocommerce
+PUPPETEER_SLOWMO=10 pnpm -- run e2e:dev --filter=woocommerce
```

Comment:
Is the double-dash needed for this to work?

Reply:
This looks like it was hit by a find/replace. I'll review/test these docs

Reply:
I think this will need the "turbo treatment" as well

Reply:
We should probably just remove `lint` and all of the `e2e` actions from `turbo.json`. This was mostly done so that you could run `turbo` commands, but, since PNPM has the filtering already, we don't get anything out of it.